catalog/transformer/catalog-transformer-propertyjson-metacard/src/main/java/ddf/catalog/transformer/metacard/propertyjson/PropertyJsonMetacardTransformer.java

```diff
@@ -0,0 +1,214 @@
+import java.nio.charset.StandardCharsets;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TimeZone;
+
+import javax.activation.MimeType;
+import javax.activation.MimeTypeParseException;
+import javax.annotation.Nullable;
+import javax.xml.bind.DatatypeConverter;
+
+import org.boon.json.JsonFactory;
+import org.boon.json.JsonParserFactory;
+import org.boon.json.JsonSerializer;
+import org.boon.json.JsonSerializerFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.AttributeType;
+import ddf.catalog.data.BinaryContent;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.BinaryContentImpl;
+import ddf.catalog.transform.CatalogTransformerException;
+import ddf.catalog.transform.MetacardTransformer;
+
+/**
+ * Implements the {@link MetacardTransformer} interface to transform a single {@link Metacard}
+ * instance to GeoJSON. This class places what is returned by {@link Metacard#getLocation()} in the
+ * geometry JSON object in the GeoJSON output. The rest of the attributes of the Metacard are placed
+ * in the properties object in the JSON. See geojson.org for the GeoJSON specification.
+ *
+ * @author Ashraf Barakat
+ * @author ddf.isgs@lmco.com
+ * @see MetacardTransformer
+ * @see Metacard
+ * @see Attribute
+ */
+
+/**
+ * Transforms a metacard into a JSON with a format similiar to geojson, but without any
+ * special geojson components. Simply put, a flat mapping of all attributes to K/V pairs
+ * stored inside a root key of "property"
+ */
+public class PropertyJsonMetacardTransformer implements MetacardTransformer {
+
+    private static final Logger LOGGER =
+            LoggerFactory.getLogger(PropertyJsonMetacardTransformer.class);
+
+    private static final String SOURCE_ID_PROPERTY = "source-id";
+
+    private static final String ISO_8601_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
+
+    private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat(ISO_8601_DATE_FORMAT);
+
+    private final JsonSerializer json = JsonFactory.create(new JsonParserFactory(),
+            new JsonSerializerFactory().includeNulls()
+                    .includeEmpty()
+                    .includeBlank()
+                    .setJsonFormatForDates(false))
+            .serializer();
+
+    protected static final String METACARD_TYPE_PROPERTY_KEY = "metacard-type";
+
+    protected static final MimeType DEFAULT_MIME_TYPE = new MimeType();
+
+    public static final String ID = "propertyjson";
+
+    static {
+        try {
+            DEFAULT_MIME_TYPE.setPrimaryType("application");
+            DEFAULT_MIME_TYPE.setSubType("json");
+        } catch (MimeTypeParseException e) {
+            LOGGER.info("Failure creating MIME type", e);
```

Comment:
TODO(Ryan) - Maybe replace with constants so we don't have to deal with exception at all